File: tests/machine/x/lua/match_expr.lua
x = 2
label = (function()
    local _t0 = x
    if _t0 == 1 then return "one" end
    if _t0 == 2 then return "two" end
    if _t0 == 3 then return "three" end
    return "unknown"
end)()
print(label)

File: tests/machine/x/scala/match_expr.scala
object match_expr {
  val x = 2
  val label = x match {
    case 1 => "one"
    case 2 => "two"
    case 3 => "three"
    case _ => "unknown"
  }
  def main(args: Array[String]): Unit = {
    println(label)
  }
}
